### DISCUSSION TITLE
New Logo

### DISCUSSION THREAD

**Discussion by hassandraga** [webui-dev/webui#292] [General]
As you may already know, when I was looking for a logo, I searched for the letter "w" (_first letter of `webui`_) in `icons8.com`, but it seems the one I randomly chose is a trademark of Waterfox Browser. So, we need to design a new logo.

If anyone has a suggestion for a new logo, please share it here, but first, please make sure it is original and not a trademark logo.

Thank you.
Regards,

**Comment by jinzhongjia** [comment]
Maybe we can directly use AI to generate one?

**Reply by hassandraga** [reply]
Sure

**Comment by hassandraga** [comment]
How about this one?

![WEBUI](https://github.com/webui-dev/webui/assets/16948659/249b6070-eed2-4669-8434-ed19d3e4dc30)

**Reply by ttytm** [reply]
I think it looks modern and has good colors. Although my choice compared to the current logo would still be the current one - but in general I think a new logo is a good idea.

To make some hopefully constructive critique why I think so: what is noticeable when you look at logos of projects that have managed to build a brand (like github, youtube, docker, nginx proxy manger) is that they usually have an icon that also works without the text. Such a symbol manages to stick to the minds of its viewers on its own - so it's really a form of a branding :sweat_smile:.

It would be good to try get one or two alternatives that try to achieve this.

**Reply by ttytm** [reply]
I want to use the opportunity to mention my recent silence. Currently, work got more busy also I got fairly ill with a myocarditis which will take me probably some months to fully recover. Unfortunately I'm not finding moments to do a lot of additional open source work.

**Reply by hassandraga** [reply]
I'm sorry to hear about your illness Turiiya. It's important to focus on your health and recovery. Take your time, and don't worry about open source. I am wishing you a quick recovery.

**Comment by fibodevy** [comment]
Some AI generated ideas

<details>
<summary>Click to expand / collapse</summary>

![8](https://github.com/webui-dev/webui/assets/21068718/28cbdf05-dad1-48dc-9b5f-44f9934ccc63)
![5](https://github.com/webui-dev/webui/assets/21068718/be53eeb6-8894-4175-823d-84d176dbf6c5)
![6](https://github.com/webui-dev/webui/assets/21068718/dde6cfd7-cdba-4e8a-b4be-c37fdfa035fe)
![3](https://github.com/webui-dev/webui/assets/21068718/79a4b6a5-2960-489c-83f8-72a8f649e4f4)
![1](https://github.com/webui-dev/webui/assets/21068718/656ad1fd-7476-4335-9abe-403e0b29c906)
![4](https://github.com/webui-dev/webui/assets/21068718/c56a4ed9-41d0-4554-8ca4-68c3e4d87f93)
![9](https://github.com/webui-dev/webui/assets/21068718/b1129b2a-cfb0-4c29-96c2-918eb6d0b6ff)
![7](https://github.com/webui-dev/webui/assets/21068718/54360389-9dfe-4b1d-bca2-557589317fb9)
![2](https://github.com/webui-dev/webui/assets/21068718/c0dfb19d-a8b2-4e1a-8c97-f5cf0546eb70)

</details>

**Reply by jinzhongjia** [reply]
I think the fifth one is good, it is simpler and the color is not too heavy

**Reply by fibodevy** [reply]
[SVG cropped one-color version](https://github.com/webui-dev/webui/assets/21068718/648f5cf5-f9ef-49d8-9238-d736431fbf74), preview:

![8bqweoM388](https://github.com/webui-dev/webui/assets/21068718/3496f174-3277-413f-b3b8-5133b7ec93b1)

**Reply by hassandraga** [reply]
Thank you for the suggestions. I have already tried using AI to generate logos, but AI logos are too colourful and complex and have a lot of details. AI logos are suitable for a screen background picture. However, that's what I noticed in the AI platforms I tested. I don't know about others.

I suggested a simpler basic logo, or as @ttytm suggested, an icon that works without the text, a symbol.

**Comment by hassandraga** [comment]
Because the original logo owners keep asking to change this free open-source project's logo and speed up the process, therefore, I will use this logo, at least for now. It simply represents the backend window and the frontend window.

![WEBUI](https://github.com/webui-dev/webui/assets/16948659/707203c2-15ff-4477-bde1-2ae7ad72b532)

**Reply by jinzhongjia** [reply]
looks ok
